docs/bounding_volume_hierarchy.md

```diff
@@ -0,0 +1,188 @@
+# `ArborX::BVH`
+Defined in header [`<ArborX_LinearBVH.hpp>`](https://github.com/arborx/ArborX/blob/master/src/ArborX_LinearBVH.hpp)
+```C++
+template <typename DeviceType>
+class BoundingVolumeHierarchy;
+```
+
+The class template `ArborX::BoundingVolumeHierarchy` is a tree data structure that can be used to accelerate the search for geometrical objects in space.
+
+ArborX defines the `ArborX::BVH` alias template for convenience.
+```C++
+template <typename DeviceType>
+using BVH = BoundingVolumeHierarchy<DeviceType>;
+```
+
+## Template parameter
+`DeviceType`
+: A valid Kokkos device type, e.g. `Kokkos::Cuda::device_type` or `Kokkos::OpenMP::device_type`.
+
+## Member types
+Member type | Definition
+--- | ---
+`device_type` | `DeviceType`
+`size_type` | `DeviceType::memory_space::size_type`
+`bounding_volume_type` | `ArborX::Box`
+
+## Member functions
+[`(constructor)`](#arborxbvhdevicetypebvh)
+: constructs the tree data structure.
+
+[`size`](#arborxbvhdevicetypesize)
+: returns the number of leaves stored in the tree.
+
+[`empty`](#arborxbvhdevicetypeempty)
+: checks whether the tree has no leaf.
+
+[`bounds`](#arborxbvhdevicetypebounds)
+: returns a bounding volume able to contain all leaves stored in the tree.
+
+[`query`](#arborxbvhdevicetypequery)
+: finds all leaves that meet some predicates, e.g. nearest to a point or intersecting with a box or a sphere.
+
+
+## Non-member functions
+TBD
+
+# `ArborX::BVH<DeviceType>::BVH`
+```C++
+BVH() noexcept; // (1)
+
+template <typename Primitives>
+BVH(Primitives const& primitives); // (2)
+```
+1) Default constructor.  Constructs an empty tree.  
+2) Constructs a bounding volume hierarchy from a variety of data source.
+## Parameters
+`primitives`
+: geometrical objects one wish to index
+#### Type requirements
+A specialization of [`ArborX::Traits::Access`](https://github.com/dalg24/ArborX/blob/docs/docs/traits.md#arborxtraitsaccess) must match `Primitives` as the first template argument and `PrimitivesTag` as second argument.  
+`ArborX::Traits::Access<Primitives,PrimitivesTag>::get()` return type must decay either to `ArborX::Point` or `ArborX::Box`.  
+ArborX provides specializations for Kokkos views but the user may specialize it for his custom types.
+
+## Complexity
+$O(N log(N))$, where `N = ArborX::Traits::Access<Primitives,PrimitivesTag>::size(primitives)` is the number of primitives passed to the constructor.
```

Comment:
Is this rendered for you? I see this literally as `$O(N log(N))$`.

Reply:
It was rendered within JupyterLab but it is not supported on GH rendered markdown.  They might be other options but I might just drop the dollar signs for now.